### PR TITLE
Added `x,y,z` layouts to the layout module.

### PR DESCRIPTION
**Pull request by antrikshmisri**
This PR adds the functionality of `x`, `y`, `z` layouts in the layout module. Please merge #480 and #479 before reviewing/merging this.

### CODE REVIEW

**Comment by skoudoro** [comment]
Can you update/rebase this PR,  then, I will start a review. thank you

**Comment by antrikshmisri** [comment]
Hey @skoudoro, this has been rebase, PTAL. Thanks!

**Review by skoudoro — COMMENTED** [review]
Hi @antrikshmisri,

Overall, it looks good. I need to test it before merging.

Can you assure that your test cover all the cases, some lines are not covered. 

See below for some comments.

File: fury/layout.py [hunk 1]
```diff
@@ -214,3 +211,263 @@ def compute_positions(self, actors):
...
+            side of the cell. If a single float is provided then both pad_x and
+            pad_y will have the same value.
+        cell_shape : {'rect', 'square', 'diagonal'} (optional)
+            Specifies the desired shape of every cell.
+            'rect' ensures the cells are the tightest.
+            'square' ensures the cells are as wide as high.
+            'diagonal' ensures the content of the cells can be rotated without
+            colliding with content of the neighboring cells.
+        """
+        self.direction = direction
```
[the remark this answers is not in the document]
Comment (antrikshmisri) [reply fury/layout.py]: Done.

File: fury/layout.py [hunk 2]
```diff
@@ -214,3 +211,263 @@ def compute_positions(self, actors):
...
```
[the remark this answers is not in the document]
Comment (antrikshmisri) [reply fury/layout.py]: Done.

File: fury/layout.py [hunk 3]
```diff
@@ -214,3 +211,263 @@ def compute_positions(self, actors):
...
```
[the remark this answers is not in the document]
Comment (antrikshmisri) [reply fury/layout.py]: Done.

**Comment by skoudoro** [comment]
it looks good and #479 #480 has been merged. Thank you @antrikshmisri. merging

**Pull request merged**